src/api/common/validate.js

```diff
@@ -25,9 +25,10 @@ function validateAddressAndSecret(obj) {
 }
 
 function validateLedgerRange(options) {
-  if (_.isUndefined(options.minLedger) && _.isUndefined(options.maxLedger)) {
-    if (Number(options.minLedger) > Number(options.maxLedger)) {
-      throw error('minLedger must not be greater than maxLedger');
+  if (_.isUndefined(options.minLedgerVersion)
```

Comment:
Looks like it happened before your change... I have a suspicion it may have been me